app/src/test/java/com/codesoom/assignment/application/TaskServiceTest.java

```diff
@@ -0,0 +1,197 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.TaskNotFoundException;
+import com.codesoom.assignment.models.Task;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+@DisplayName("TaskService 클래스의")
+class TaskServiceTest {
+    private TaskService taskService;
+
+    @BeforeEach
+    void setUp() {
+        // subject
+        taskService = new TaskService();
+    }
+
+    @Nested
+    @DisplayName("getTasks 메서드는")
+    class Describe_getTasks {
+
+        @Nested
+        @DisplayName("만약 1개의 할 일이 등록되어 있다면")
+        class Context_with_one_task {
+            // fixture
+            private final String validTaskTitle = "test";
+
+            @BeforeEach
+            void prepareTasks() {
+                // fixture
+                Task task = new Task();
+                task.setTitle(validTaskTitle);
+                taskService.createTask(task);
+            }
+
+            @Test
+            @DisplayName("등록된 할 일 1개를 반환한다")
+            void It_returns_one_task() {
+                List<Task> tasks = taskService.getTasks();
+                assertAll(
+                        () -> assertThat(tasks).hasSize(1),
+                        () -> assertThat(tasks.get(0)
+                                              .getTitle()).isEqualTo(validTaskTitle));
+            }
+        }
+    }
+
+    @Nested
+    @DisplayName("getTask 메서드는")
+    class Describe_getTask {
+
+        @Nested
+        @DisplayName("만약 기존에 등록된 할 일을 조회한다면")
```

Comment:
`기존에`는 시간에 의존하는 단어이기 때문에 프로그래밍에서는 주석이건 코드에서건 가능한 한 사용하지 않는 것이 좋습니다.
주석으로 `기존에...`라고 적었는데, 그 다음주에 다른 사람도 다른 코드에 `기존에...`라는 주석을 작성해 남겼다고 생각해 봅시다. 두 `기존에...`는 같은 시각을 지칭하지 않습니다.

다른 좋은 표현도 많습니다.

```suggestion
        @DisplayName("만약 등록되어 있는 할 일을 조회한다면")
```